src/components/Footer/Footer.css

```diff
@@ -0,0 +1,34 @@
+.footer{
+    background-color: var(--footer-bg);
+    color: white;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    transition:  0.8s;
+}
+
+.footer-text{
+    font-size: 16px;
+    font-weight: 600;
+    margin: 10px 0 8px 0;
+}
+
+.code-icn{
+    color: #aba8a8;
```

Comment:
Try to add this as global color variable

Reply:
yes will add it to global 👍